crowbar_framework/app/assets/javascripts/barclamps/neutron/application.js

```diff
@@ -253,26 +253,52 @@ function networking_plugin_check() {
 }
 
 function ml2_type_drivers_check() {
-  var values = $('#ml2_type_drivers').val();
+  var values = $('#ml2_type_drivers').val() || [];
   if (values.indexOf("vlan") >= 0) {
     $('#num_vlans_container').show();
   } else {
     $('#num_vlans_container').hide();
   }
+
+  if (values.length == 1) {
```

Comment:
This logic doesn't work. When doing the following, I get ```The default provider network type driver "vlan" is not a selected ml2 type driver ``` as error.
* select both ml2 type drivers
* select for both default values the second driver from the ```ml2 type drivers``` select field
* now select only the first driver in the ```ml2 type drivers``` select field
* press save - here's the error message